src/Plugin/views/filter/InOperator.php

```diff
@@ -137,4 +137,44 @@ protected function opSimple() {
     }
   }
 
+  public function operators() {
+    $operators = parent::operators();
+    $operators += [
```

Comment:
Consider wrapping this in    ` if (!empty($this->definition['allow empty'])) {`

Reply:
nice